### PR TITLE
[CELEBORN-1240][FOLLOWUP] Introduce web profile for web module

### PR DESCRIPTION
**Pull request by SteNicholas**
### What changes were proposed in this pull request?

Introduce web profile for web module.

### Why are the changes needed?

The compilation speed of web module is sometimes very slow due to the influence of the network, which hinders the development process.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

CI.

### CODE REVIEW

**Comment by FMX** [comment]
Thanks. There is some changes to be done in the sbt build script.

**Comment by SteNicholas** [comment]
@FMX, the web module of the sbt build script will support in anther pull request.

**Comment by SteNicholas** [comment]
@FMX, PTAL.

File: project/CelebornBuild.scala [hunk 1]
```diff
@@ -1432,3 +1433,8 @@ object CelebornOpenApi {
     pomPostProcess := removeDependenciesTransformer
   )
 }
+
+object CelebornWeb {
```
Comment (FMX) [inline project/CelebornBuild.scala:1437]: Web module should not be active by default.  maybeWebModules will be suitable.

**Comment by SteNicholas** [comment]
@FMX, thanks for review. I have addressed above comment and actived the web module in sbt. PTAL.

**Review by FMX — APPROVED** [review]
LGTM. Merged into main(v0.6.0).

**Pull request closed without merging**